### PR TITLE
Add test to check invariant qualifier doesn't leak state

### PR DESCRIPTION
**Pull request by qkmiao**

### CODE REVIEW

**Comment by kenrussell** [comment]
Aren't these cases already covered in sdk/tests/conformance/glsl/misc/shaders-with-invariance.html ?

**Comment by kenrussell** [comment]
Sorry for my confusion; I just saw your https://chromium-review.googlesource.com/366720 which fixes bugs in this area.

I continue to wonder whether sdk/tests/conformance/glsl/misc/shaders-with-invariance.html already covers this case.

**Comment by qkmiao** [comment]
Actually, these tests in this pr are from sdk/tests/conformance/glsl/misc/shaders-with-invariance.html, but in different orders. This pr moves the test which has invariant qualifier varying before the test which doesn't have invariant qualifier. So, the bug happens.

File: sdk/tests/conformance/glsl/bugs/invariant-does-not-leak-across-shaders.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+varying vec4 v_varying;
+
+void main() {
+    gl_Position = v_varying;
+}
+</script>
+<script type="text/javascript">
+"use strict";
+description("The use of the invariant qualifier in one shader must not affect other shaders.");
+
```
Comment (kenrussell) [inline sdk/tests/conformance/glsl/bugs/invariant-does-not-leak-across-shaders.html]: Could you add a debug() call here, mentioning that this is a deliberate subset of glsl/misc/shaders-with-invariance.html, and linking to http://crbug.com/634813 ? Thanks.

**Comment by kenrussell** [comment]
One minor request. Could you please add it and comment on this pull request when done? Thanks.

**Comment by qkmiao** [comment]
Thanks for review. debug message was added in the new pr.

**Comment by kenrussell** [comment]
Thank you. LGTM.

**Pull request merged**